core/src/main/java/com/onelogin/saml2/Auth.java

```diff
@@ -95,6 +97,16 @@
 	private String errorReason;
 
 	/**
+	 * The id of the last AuthnRequest generated
+	 */
+	private String lastLoginRequestId;
```

Comment:
Do you think is necessary to uselastLoginRequestId lastLogoutRequestId rather than just _lastRequestID?

When you are using the auth class, you already control what methods do you call, so I think we can manage all in 1 unique parameter.